cli/util/uploader.go

```diff
@@ -411,6 +412,13 @@ func updateIndexFiles(ctx *cli.Context, p *pool.Pool, containerID cid.ID, accoun
 				}
 			}
 		}
+		emptyOid := make([]byte, 32)
```

Comment:
Move it to some global variable in order not to allocate it every time. Replace `32` with proper constant. Write relevant comment.